content/en/sidecar-container.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Sidecar Container
+status: Feedback Appreciated
+category: concept
+---
+
+The Sidecar Container is an implementation of the sidecar pattern. 
+An application deployed on a second container will run side by side 
+and share its lifecycle with a primary application running on a main container. 
+
+## Problem it addresses
+
+There are scenarios where it is convenient to group [containers](/container/) 
+to deal with cross-platform requirements or shared functionality.
+Such scenarios include [observability](/observability/) topics 
+like logging, monitoring, tracing, but also security, and networking. 
+For example, when multiple microservices have their sidecar containers 
```

Comment:
Here we talk already about the solution. This would be where you should show how collecting logs would look without the sidecar container.